scene/main/viewport.cpp

```diff
@@ -1222,6 +1222,21 @@ void Viewport::_propagate_viewport_notification(Node *p_node, int p_what) {
 	}
 }
 
+void Viewport::force_can_drop_data_check() {
+	DisplayServer::CursorShape ds_cursor_shape = (DisplayServer::CursorShape)Input::get_singleton()->get_default_cursor_shape();
```

Comment:
```suggestion
	DisplayServer::CursorShape ds_cursor_shape;
```

This doesn't need a default value as it will be set later.